README.md

```diff
@@ -38,6 +38,7 @@ The following tasks are no longer being developed and will be deprecated in a fu
 | Task Name | Alternative |
 |-----------|-------------|
 | st0236_set_cache_paths_to_default | Use [puppet conf](https://forge.puppet.com/modules/puppetlabs/puppet_conf/readme) |
+| st0305_support_script_and_upload | MFT and P1 are the preferred methods for upload | 
```

Comment:
This should link to the [kb article](https://perforce1--uat.sandbox.my.site.com/Community2/s/article/360009970114) for MFT uploads.